### PR TITLE
Minor Issues and Update Allowed Types

### LINKED ISSUE #76: Upload Endpoint
Make upload endpoint work with different types.
For instance video etc.

### LINKED ISSUE #81: Make Sphinx a development dependency only

### LINKED ISSUE #87: Import Order
Standard import "import os" should be placed before relative imports in project template...

### LINKED ISSUE #89: PipelineWrapper has no attribute 'get_config'
Screen Shot 2019-03-22 at 2 56 18 pm

### PR DESCRIPTION
**Pull request by AkshatBajaj**
Fixes #76 
Fixes #81
Fixes #87 
Fixes #89 

 - Check MIME type of JSON and Image
 - Add support for video

### CODE REVIEW

File: surround/surround.py [hunk 1]
```diff
@@ -118,24 +119,38 @@ def process(self, surround_data):
...
         self.surround = surround
+        self.actual_type_of_uploaded_object = None
         if type_of_uploaded_object:
             self.type_of_uploaded_object = type_of_uploaded_object
         else:
             self.type_of_uploaded_object = AllowedTypes.JSON
         self.surround.init_stages()
 
     def run(self, input_data):
+        mime = magic.Magic(mime=True)
```
Comment (ScottyB) [inline surround/surround.py]: Why use Magic?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 pyaml==17.12.1
 doit==0.31.1
 Sphinx==1.8.4
```
Comment (ScottyB) [inline requirements.txt]: Sphinx can be removed.

**Comment by AkshatBajaj** [comment]
- Remove video support
- Remove Sphinx
- Get version dynamically

**Comment by ScottyB** [comment]
After installing Surround I get the error below. Can you find another dependency? The `python-magic` package requires the user to install other dependencies manually and I'd like to avoid this overhead if possible.
```
➜  temp git:(pr/88) ✗ surround init
Traceback (most recent call last):
  File "/usr/local/bin/surround", line 11, in <module>
    load_entry_point('surround==0.0.0', 'console_scripts', 'surround')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2691, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2322, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2328, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.6/site-packages/surround-0.0.0-py3.6.egg/surround/__init__.py", line 1, in <module>
    from .surround import Surround, SurroundData, Wrapper, AllowedTypes
  File "/usr/local/lib/python3.6/site-packages/surround-0.0.0-py3.6.egg/surround/surround.py", line 11, in <module>
    import magic
  File "/usr/local/lib/python3.6/site-packages/python_magic-0.4.15-py3.6.egg/magic.py", line 181, in <module>
    raise ImportError('failed to find libmagic.  Check your installation')
ImportError: failed to find libmagic.  Check your installation
```

```
➜  temp git:(pr/88) ✗ pip3 freeze | grep magic
python-magic==0.4.15
```

**Comment by AkshatBajaj** [comment]
- python-magic removed

**Pull request merged**